website/templates/project/project_base.mako

```diff
@@ -14,6 +14,11 @@
 
 <%include file="project_header.mako"/>
 
+% if status and node:
+    ## WARNING: Need new heading alert messages? They must be added at both here and base.mako
+    <%include file="../alert.mako"/>
```

Comment:
I was referring to the mako context--not Javascript. Also, you are using `node` in `base.mako`, which is inherited by every page (not just project pages).